### PR TITLE
Fix wxGUI Graphical Modeler decode string

### PR DESCRIPTION
**Pull request by tmszi**
Fix decode string.

### CODE REVIEW

File: gui/wxpython/gmodeler/model.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def LoadModel(self, filename):
         try:
             gxmXml = ProcessModelFile(etree.parse(filename))
         except Exception as e:
-            raise GException(unicode(e))
+            raise GException(grass.decode(e))
```
Comment (landam) [inline gui/wxpython/gmodeler/model.py]: There seems to be no need to encode/decode. 

```
raise GException('{}'.format(e))
```

is enough.

File: gui/wxpython/gmodeler/model.py [hunk 2]
```diff
@@ -653,7 +653,7 @@ def Run(self, log, onDone, parent=None):
             delInterData = dlg.DeleteIntermediateData()
             dlg.Destroy()
             if err:
-                GError(parent=parent, message=unicode('\n'.join(err)))
+                GError(parent=parent, message=grass.decode('\n'.join(err)))
```
Comment (landam) [inline gui/wxpython/gmodeler/model.py]: There seems to be no need to encode/decode.

```
GError(parent=parent, message='\n'.join(err))
```

is enough.

Comment (landam) [reply gui/wxpython/gmodeler/model.py]: `task.get_cmd_error()` returns list of strings.

File: gui/wxpython/gmodeler/model.py [hunk 3]
```diff
@@ -669,7 +669,7 @@ def Run(self, log, onDone, parent=None):
                     parent=parent,
                     message=_("Variables below not defined:") +
                     "\n\n" +
-                    unicode(
+                    grass.decode(
```
Comment (landam) [inline gui/wxpython/gmodeler/model.py]: There seems to be no need to encode/decode. `err` contains tuple of strings.

**Comment by tmszi** [comment]
Reproduce:

1. Open Graphical Modeler
2. Create simple model (convert vector map to the raster, `v.to.rast` command )
3. On the `v.to.rast`  dialog set 
3.1. Name of input vector map (Required tab)
3.2. Name of output raster map (Required tab)
3.3. Source of raster values (Required tab): **cat**
3.4. Category values (Selection tab): check checkbox **Parametrized in model**
4. Run Model
5. Do not set Category values in the open dialog

I got error message in the **Console** page:

```
Traceback (most recent call last):
  File "/usr/local/grass79/gui/wxpython/gmodeler/frame.py",
line 623, in OnRunModel

self.model.Run(self._gconsole, self.OnModelDone,
parent=self)
  File "/usr/local/grass79/gui/wxpython/gmodeler/model.py",
line 672, in Run

unicode(
NameError
:
name 'unicode' is not defined
```

File: gui/wxpython/gmodeler/model.py (hunk 3 above)
Comment (tmszi) [reply gui/wxpython/gmodeler/model.py]: Alright of course, I remove this function.

**Pull request merged**